test/cpp/jit/test_gpu.cpp

```diff
@@ -1,4 +1,4 @@
-// #if defined(USE_CUDA)
+#if defined(USE_CUDA)
```

Comment:
Is this change related to this fix?

Reply:
no, think I commented it in the last PR and forgot to change it back.